tajo-storage/tajo-storage-pgsql/pom.xml

```diff
@@ -236,7 +236,6 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
```

Comment:
It is my fault. but your patch can't run tests
please override configuration as follows:
<configuration combine.self="override">